pages/api/studentPortal/units.ts

```diff
@@ -54,6 +54,23 @@ export const reactUnits: Unit[] = [
       },
       {
         title: "Lesson 3",
+        description: "TailwindCSS - Grid & Flexbox",
```

Comment:
I think it might make sense to put this series of lessons before the React ones since I kind of expect you to know a bit of tailwind to style a table